packages/notifications/src/Notification.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace Filament\Notifications;
+
+use Filament\Support\Components\ViewComponent;
+use Livewire\Wireable;
+
+class Notification extends ViewComponent implements Wireable
+{
+    protected string $view = 'notifications::components.notification';
+
+    public string $id;
+
+    public bool $closed = false;
+
+    public string $title;
+
+    public ?string $description = null;
+
+    public ?string $icon = null;
+
+    public ?string $status = null;
+
+    public function __construct()
+    {
+        $this->id(uniqid());
+    }
+
+    public static function make(): self
```

Comment:
All `self` types in these files can be switched to `static`

Reply:
Good one! 9a8c19fe4f3dc59125b935898b7ad6ef279f4a21